src/paths/multigrid.jl

```diff
@@ -0,0 +1,26 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    MultiGridPath()
+
+to-do
+"""
+struct MultiGridPath <: Path end
+
+function traverse(grid::Grid{Dim}, ::MultiGridPath) where {Dim}
+  path = Int[]
+  sizes = size(grid)
```

Comment:
I think `dims` is betrer than `sizes`. What do you think?

Reply:
Yes, agree here.